lib/Common/Memory/AllocatorTelemetryStats.h

```diff
@@ -0,0 +1,25 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include "Common/Tick.h"
+
+struct AllocatorDecommitStats
+{
+    Js::Tick lastLeaveDecommitRegion;
```

Comment:
What does region refer to in lastLeaveDecommitRegion? I would prefer lastLeaveDecommitTime?

Reply:
per conversation w/ @curtisman, the "decommit region" is the period between `EnterIdelDecommit` and `LeaveIdelDecommit`.  My understanding here during this period, "idle decommit" is suspended.  Curtis was interested in understanding the period of time between when we left this "idle decommit suspended region" and when the idle decommit occurs. 

Names here are a bit confusing - it's possible I've got this wrong so lmk if above doesn't make sense.